src/wp-includes/comment-template.php

```diff
@@ -2467,6 +2468,7 @@ function wp_list_comments( $args = array(), $comments = null ) {
  *                                        Default 'Your email address will not be published.'.
  *     @type string $comment_notes_after  HTML element for a message displayed after the textarea field.
  *     @type string $action               The comment form element action attribute. Default '/wp-comments-post.php'.
+ *     @type bool   $novalidate_form      Whether the novalidate attribute is added to the comment from. Default false.
```

Comment:
```suggestion
 *     @type bool   $novalidate_form      Whether the novalidate attribute is added to the comment form. Default false.
```

simple typo correction